github.go

```diff
@@ -13,10 +13,18 @@ import (
 	"strings"
 
 	"github.com/google/go-github/github"
+	"github.com/gregjones/httpcache"
+	"github.com/gregjones/httpcache/diskcache"
 	"github.com/shurcooL/githubv4"
 	"golang.org/x/oauth2"
 )
 
+// on-disk location to store cached API results in between invocations
+// primarily useful for the Check command, which will be re-run in the same container multiple times
+// the underlying disk caching library automatically finds-or-creates this directory.
+// if the given path can't be created as a writeable directory, the whole thing will silently fail to cache.
+const diskCachePath = "/tmp/github-api-cache"
```

Comment:
Use https://golang.org/pkg/io/ioutil/#TempDir?

Reply:
in the case of `ioutil.TempDir` - this function intentionally creates a randomly named directory, intended to be globally unique. This means that two runs won't use the same cache directory, bypassing the point.

TempDir basically returns `/tmp`, which maybe makes constructing a shared cache path a bit more reasonable in a cross-platform way, but there's no guarantee that the directory exists or is writeable.

Reply:
`os.TempDir` should be OS agnostic.  I would assume the check container runs in privilege mode?